app/test/meadow/config_test.exs

```diff
@@ -90,15 +90,15 @@ defmodule Meadow.ConfigTest do
     end
 
     test "trailing slashes/0" do
-      Application.put_env(:meadow, :iiif_server_url, "http://no-slash-test/iiif/2")
+      Application.put_env(:meadow, :iiif_server_url, "http://no-slash-test/3")
```

Comment:
Is there a `iiif/` missing here? And below?

Reply:
Thanks for catching that! Pushed a change.